packages/salesforcedx-vscode-soql/test/vscode-integration/client/completion.test.ts

```diff
@@ -67,7 +67,7 @@ workspace.registerTextDocumentContentProvider('embedded-soql', {
       .then(content => content.toLocaleString());
   }
 });
-const configDir = '.sfdx';
+const configDir = '.sf';
```

Comment:
Revert this change - the test fails when you change the `configDir` to `.sf`.

I think the test failure is because we are still keeping `sfdx-config.json` in the `.sfdx` folder as our main config file.